spec/test-outputs/www-integration.out.vcl

```diff
@@ -426,6 +454,32 @@ sub vcl_deliver {
   }
   # End dynamic section
 
+if (req.http.Cookie ~ "cookies_policy") {
+  if (req.http.Cookie:cookies_policy ~ "%22usage%22:true" && req.http.User-Agent !~ "^GOV\.UK Crawler Worker") {
```

Comment:
I'm curious both here and in the previous code in the receive block why we're not checking to ensure that the test is active as we do with the other tests? Ideally we should do this as a means to be able to control the test through configuration.

Reply:
True, we can set all the values in the `start_a_business_ab_test_base_paths.yaml` to false which should have the same effect but happy to add it if it makes things a bit more standard

Reply:
Cool. It'd be good to do that, not least if you need to turn it off for any reasons and have to update all of those values rather than a simple switch 🙂